db/migrate/20190915142043_migrate_run_video_urls_to_videos.rb

```diff
@@ -0,0 +1,22 @@
+class MigrateRunVideoUrlsToVideos < ActiveRecord::Migration[6.0]
+  # video_url is included in ignored_columns in Run, so we can't use standard ActiveRecord stuff to create Videos or rollback the migration
+  def up
+    sql = "INSERT INTO videos (run_id, url, created_at, updated_at)
+           SELECT id, video_url, now(), now()
+           FROM runs
+           WHERE
+             video_url IS NOT NULL
+             AND video_url <> ''
+          ".squish
```

Comment:
I'd actually slightly prefer that we don't carry over `created_at` and `updated_at`, as they might be wrong when talking strictly about when the video was added.

Reply:
They're not being carried over. Both fields are being set to the current time on the DB server.

Reply:
Oh sorry about that, I misread.